google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -468,10 +469,10 @@ public Builder setCredentialsProvider(CredentialsProvider credentialsProvider) {
     }
 
     /**
-     * Gives the ability to set a custom executor for managing lease extensions. If none is provided
-     * a shared one will be used by all {@link Subscriber} instances.
+     * Gives the ability to set a custom executor for polling and managing lease extensions. If none
+     * is provided a shared one will be used by all {@link Subscriber} instances.
      */
-    public Builder setLeaseAlarmsExecutorProvider(ExecutorProvider executorProvider) {
+    public Builder setSystemExecutorProvider(ExecutorProvider executorProvider) {
```

Comment:
I mainly went with this based on [David's suggestion](https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2079#discussion_r118393132). Feels suboptimal, but I struggle to think of a better name.

The way I think about it: the "time" command gives you "user" and "system" time, vaguely split into (1) "what you want to do" and (2) "whatever the kernel needs to do so you can do 1". To us, it's (1) "what you want to do with the messages" and (2) "whatever we need to do so you can do 1". The word "system" is vague because the user doesn't generally need to care whatever it is that we do, the same way I don't generally care what the kernel does to read me the file.

Am I making any sense?